src/python_bindings/bindings.cpp

```diff
@@ -305,6 +305,9 @@ class PyHNSWLibIndex : public PyVecSimIndex {
         } else if (type == VecSimType_INT8) {
             auto *hnsw = dynamic_cast<HNSWIndex<int8_t, float> *>(index.get());
             hnsw->saveIndex(location);
+        } else if (type == VecSimType_UINT8) {
```

Comment:
i can't comment at the relevant location :unamused:
why did you remove `VecSimType_INT8` from `getVector(labelType label)` (instead of adding `VecSimType_UINT8`)